resources/assets/js/navbar.js

```diff
@@ -47,7 +47,9 @@ const Navbar = {
                 if (this.invertOnScroll) {
                     if (progress === 1) {
                         this.header.classList.add("inverted");
+                        this.header.classList.remove("not-inverted");
                     } else {
+                        this.header.classList.add("not-inverted");
```

Comment:
isn't `not-inverted` the same as having a default styling that gets overwritten by `inverted:`? 🤔

Reply:
no because in this case the homepage would have black when not inverted (scrolled to the top). It needs a different colour: `inverted:text-theme-secondary-900 not-inverted:text-theme-primary-100 text-theme-secondary-900` on the hamburger. not-inverted is only applied to the navbar when inverting is possible, navbars that don't have that will take the default. I can probably remove the `inverted:text-theme-secondary-900` though :thinking:

Reply:
removed that `inverted:text-theme-secondary-900` class as it's the same as the base colour